resources/sample_files/README.md

```diff
@@ -0,0 +1,5 @@
+## Sample Files
+These files are included as shared sample data files for use in examples in this repository.
+
+* [movies.json](movies.json) `A sample data set for examples using a common movies data model.`
+* [speech_sample.mp3](.media/speech_sample.mp3) `A sample audio file of speech used for transcribe examples.`
```

Comment:
Should "transcribe" be "transcription" or "Amazon Transcribe"?

Reply:
Fixed.